src/python/circllhist/__init__.py

```diff
@@ -0,0 +1,77 @@
+import circllhist.ffi as ffi
+
```

Comment:
Add `__all__ = ("circllbin", "circllhist")` here to not export our `ffi` definitions.